src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1865,7 +1875,20 @@ namespace winrt::TerminalApp::implementation
                 return;
             }
 
-            TermControl newControl{ *(winrt::get_self<TerminalSettings>(controlSettings)->CreateChild()), controlConnection };
+            // Give term control a child of the settings so that any overrides go in the child
+            // This way, when we do a settings reload we just update the parent and the overrides remain
+            const auto child = (winrt::get_self<TerminalSettings>(controlSettings)->CreateChild());
+            TermControl newControl{ *child, controlConnection };
+
+            // Check if the profile defines an UnfocusedAppearance
+            const auto profile = _settings.FindProfile(_settings.GetProfileForArgs(newTerminalArgs));
+            if (profile.UnfocusedAppearance())
+            {
+                // Now, make a grandchild for the unfocused appearance
+                const auto grandchild = child->CreateChild();
+                grandchild->ApplyAppearanceSettings(profile.UnfocusedAppearance(), _settings.GlobalSettings().ColorSchemes());
+                newControl.UnfocusedAppearance(*grandchild);
+            }
```

Comment:
Yeah, unfortunately it has to be this way because the Control cannot do the inheritance stuff (i.e. creating a child) because it does not have direct knowledge about `TerminalSettings`

Reply:
This has been resolved as per our discussion on teams!